CMakeLists.txt

```diff
@@ -119,6 +119,8 @@ add_definitions(
     -D__STDC_FORMAT_MACROS   # Enables printf format macros for variable sized types (e.g. size_t)
 )
 
+add_compile_definitions(_DISABLE_CONSTEXPR_MUTEX_CONSTRUCTOR)
```

Comment:
This is for Windows only. And it would be a good idea to add some comment why we are doing this.